src/encryption.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-unused-vars */
+
+export function addSkykey(portalUrl, skykey) {
+  throw new Error("Unimplemented");
+}
+
+export function createSkykey(portalUrl, skykeyName, skykeyType) {
```

Comment:
REQ: how do we handle authentication?

Reply:
There's going to be an option to pass in an API key. I'm planning to add that after this PR.